lib/pay_web/resolvers.ex

```diff
@@ -105,6 +105,10 @@ defmodule PayWeb.Resolvers do
     {:ok, Payments.get_payment_by_external_id!(external_id)}
   end
 
+  def payment(_parent, %{id: external_id}, _resolution) do
```

Comment:
this will be shadowed by the definition above. did you mean for it to be `def payment_refund(...)`?

don't see it being used

Reply:
Forgot to clean up the code. Thanks for pointing out :) 
I will remove this piece of code.